homeassistant/components/venstar/sensor.py

```diff
@@ -210,3 +229,16 @@ def native_unit_of_measurement(self) -> str | None:
     value_fn=lambda coordinator, sensor_name: coordinator.runtimes[-1][sensor_name],
     name_fn=lambda coordinator, sensor_name: f"{coordinator.client.name} {RUNTIME_ATTRIBUTES[sensor_name]} Runtime",
 )
+
+INFO_ENTITIES: tuple[VenstarSensorEntityDescription, ...] = (
+    VenstarSensorEntityDescription(
+        key="schedulepart",
+        device_class=SensorDeviceClass.ENUM,
+        options=list(SCHEDULE_PARTS.values()),
```

Comment:
The options should be lowercase snake_case and have a translation via the `translation_key`. See eg:

https://github.com/home-assistant/core/blob/49b1d6e7fef780e77afe48cdc9b4c5155c9d7703/homeassistant/components/airvisual/sensor.py#L41-L55

https://github.com/home-assistant/core/blob/49b1d6e7fef780e77afe48cdc9b4c5155c9d7703/homeassistant/components/airvisual/strings.json#L67-L76